main.go

```diff
@@ -73,6 +76,10 @@ func init() {
 	flag.Var(&includeConfigMapsDataGlobs, "configmaps-include-glob", "Configmap globs to include when looking for configmap data keys (Default \"*\").")
 	flag.Var(&excludeConfigMapsDataGlobs, "configmaps-exclude-glob", "Configmap globs to exclude when looking for configmap data keys.")
 
+	flag.BoolVar(&webhookCheckEnabled, "enable-webhook-cert-check", false, "Enable webhook cert check.")
```

Comment:
instead of an explicit enabled flag can we just check for webhooks if either label or annotation selector is set?

Reply:
If we want to check all the webhooks' certs in cluster, it's so much effort to label or annotate every single webhook. In this case, I prefer to add an enabled flag to check all the webhooks by default. WDYT?

Reply:
`--enable-webhook-cert-check=true` => check all the webhooks by default.
`--enable-webhook-cert-check=true --webhooks-label-selector=xxx` => check webhooks with label selector
`nil` => don't check webhooks by default.

Reply:
@joe-elliott PTAL, thanks.

Reply:
Ah, understood. I agree that just being able to set the enable flag if you want all webhook certs checked is desirable.